debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/MongoDbStreamingChangeEventSource.java

```diff
@@ -402,6 +401,15 @@ protected MongoDbOffsetContext initializeOffsets(MongoDbConnectorConfig connecto
         return new MongoDbOffsetContext(new SourceInfo(connectorConfig), new TransactionContext(), positions);
     }
 
+    private static String getTransactionId(Document event) {
+        final Long operationId = event.getLong(SourceInfo.OPERATION_ID);
```

Comment:
Above (in getTransactionId()) the new field takes precedence. IIUC, the fields are mutually exclusive, but let's align on one way or the other consistently.

Reply:
Btw. how does `txNumber` relate/compare to `ord`? It seems like it's kinda the same?

Reply:
I would say that `txnNumber` more closely related to `tord` rather than `ord` because the former represents the position within a transaction in which the entry took place.

Reply:
@Naros, so what's the difference between `ord` and `tord` then? I didn't even know there'd be those two :)